integreat_cms/api_client_firebase/apps.py

```diff
@@ -0,0 +1,54 @@
+"""
+Configuration of PushNotificationSender API app
+"""
+import json
+import logging
+import requests
+
+
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class PushNotificationSenderApiConfig(AppConfig):
+    """
+    PushNotificationSender API config inheriting the django AppConfig
+    """
+
+    name = "integreat_cms.push_notification_sender_api"
+    api_available = False
+
+    def ready(self):
+        """
+        Checking if API is available
+        """
+        # Only check availability if running a server
+        if apps.get_app_config("cms").test_external_apis:
+            if settings.FCM_ENABLED:
+                try:
+                    response = requests.get(
+                        f"{settings.FCM_URL}/api/", timeout=3
+                    )
+                    # Require the response to return a valid JSON, otherwise it's probably an error
+                    assert json.loads(response.text)
```

Comment:
This availability check does not make sense.
https://fcm.googleapis.com/fcm/send/api is no valid endpoint of the firebase API, so wen cannot use it to check the availability.
Either you find a more suitable endpoint for testing, or we just skip the availability test - see my comment above, I think even if it's not available for a short time, it will probably be up again and just because it had a short down time during server start does not mean we shouldn't try to send push notifications...